README.md

```diff
@@ -17,9 +17,9 @@ Add your answers to this markdown file.
 
 **1. Asymptotic analysis disregards constants, which can make considerable differences. One of these constant factors could be the machine being utilized - a supercomputer might run an asymptotically-slow algorithm faster than a tablet from 2010 could run an asymptotically-fast algorithm.**
 
-**2. Perhaps another constant factor which makes asymptotic analysis misleading would be that the algorithm has hidden dependencies - it relies on external factors that are not accounted for in the asymptotic complexity. This brings overhead which impacts the algorithm's performance, causing the actual performance to differ as a result.**
+**2. Perhaps another constant factor which makes asymptotic analysis misleading would be that the algorithm has hidden dependencies - it relies on external factors that are not accounted for in the asymptotic complexity. This brings overhead which impacts the algorithm's performance, causing the actual performance to differ as a result. An example of this could be that an algorithm fetches data from a remote server, which means it is prone to experience slowdown via latency or lack of availability if the server has a heavy load.**
```

Comment:
But asymptotic complexity only tells us how an algorithm scales. Even if it does fetch data over the internet, it would still scale the same.